timemachine/lib/potentials.py

```diff
@@ -217,17 +217,27 @@ def get_b_idxs(self):
         return self.args[1]
 
 
-class NonbondedImplWrapper(custom_ops.FanoutSummedPotential):
+class NonbondedImplWrapper:
+    # NOTE: not implemented as a subclass of
+    # custom_ops.FanoutSummedPotential to allow import of
+    # timemachine.lib.potentials to succeed when the CUDA extension is
+    # not present
+    def __init__(self, *args):
+        self._impl = custom_ops.FanoutSummedPotential(*args)
```

Comment:
Again, a bit of a hack. An alternative might be to define

```python
class FanoutSummedPotential:
    pass
```
in `custom_ops.py`, but that seemed like it might lead to more confusing errors if the user tries to use without the CUDA extension.